### PR TITLE
Trigger automatic rebase on issue comment

### PR DESCRIPTION
**Pull request by fkorotkov**
# Before submitting

- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure to update the docs?   
- [x] Did you write any new necessary tests?  
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## What does this PR do?

Instead of `pull_request` event (created, closed, etc.).

Fixes https://github.com/cirrus-actions/rebase/issues/43

## PR review    

@Borda PTAL

## Did you have fun?

Yes!

### CODE REVIEW

**Comment by awaelchli** [comment]
will this create extra notifications?

**Comment by Borda** [comment]
I think that I already did this fix somewhere, but thx!

> will this create extra notifications?

@fkorotkov can you confirm this ^^

**Comment by fkorotkov** [comment]
They constantly change something in Actions but [I just verified](https://github.com/fkorotkov/actions-demo/pull/23#issuecomment-622524988) that rebase won't appear on PRs so no noise there.

File: .github/workflows/rebase.yml [hunk 1]
```diff
@@ -17,10 +18,3 @@ jobs:
       uses: cirrus-actions/rebase@1.2
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  # https://github.community/t5/GitHub-Actions/Workflow-is-failing-if-no-job-can-be-ran-due-to-condition/m-p/38186#M3250
-  always_job:
-    name: Always run job
-    runs-on: ubuntu-latest
-    steps:
-      - name: Always run
-        run: echo "This job is used to prevent the workflow to fail when all other jobs are skipped."
```
Comment (awaelchli) [inline .github/workflows/rebase.yml:26]: So I take it the issue in the commented link is solved?

**Pull request merged**